docs/development/affiliated-packages.rst

```diff
@@ -430,7 +430,8 @@ files manually`_ section since this explains what many of the files do.
 Updating to the latest template files
 -------------------------------------
 
-.. TODO
+See instructions in Item 14 above.
```

Comment:
:+1:  Sounds good. 
Two minor points: I can't see why the numbering changed from 12 (in the current online docs) to 14? Maybe it would be more fail safe to line number changes is to add a reference to line ``"Then, each time you want to pull in changes..."``